packages/clay-card/src/Body.tsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import React from 'react';
+
+import classNames from 'classnames';
+
+import Context from './Context';
+
+export interface CardBodyProps extends React.HTMLAttributes<HTMLDivElement> {
+	children:
+		| React.ReactElement<HTMLElement>[]
+		| React.ReactElement<HTMLElement>;
+}
+
+export const Body: React.FunctionComponent<CardBodyProps> = ({
+	children,
+	className,
+	...otherProps
+}) => {
+	const {horizontal, interactive} = React.useContext(Context);
+
+	const isCardType = {
+		horizontal,
+		interactive,
+	};
+
+	const TagName =
+		isCardType.interactive ||
+		(isCardType.horizontal && isCardType.interactive)
+			? 'span'
+			: 'div';
+
+	if (isCardType.horizontal && !isCardType.interactive) {
+		return (
+			<div className="card card-horizontal">
+				<div
+					className={classNames('card-body', className)}
+					{...otherProps}
+				>
+					<div className="card-row">{children}</div>
+				</div>
+			</div>
+		);
+	}
+
+	return (
+		<TagName className={classNames('card-body', className)} {...otherProps}>
```

Comment:
I think it's to simplify and take advantage of this content here with the above condition to remove duplication of markup, we can do something like this:

> I have not tested for all use cases, but I think that's enough.

```javascript
const Content = ({children}) => (
  <TagName className={classNames('card-body', className)} {...otherProps}>
    {isCardType.interactive && !isCardType.horizontal ? children : (
      <TagName className="card-row">{children}</TagName>
    )}
  </TagName>
);

if (isCardType.horizontal && !isCardType.interactive) {
  return (
    <div className="card card-horizontal">
      <Content>
        {children}
      </Content>
    </div>
  );
}

return (
  <Content>
    {children}
  </Content>
);
```